x-pack/solutions/security/plugins/security_solution/server/lib/detection_engine/rule_management/logic/detection_rules_client/detection_rules_client.create_custom_rule.test.ts

```diff
@@ -53,7 +53,7 @@ describe('DetectionRulesClient.createCustomRule', () => {
       mlAuthz,
       savedObjectsClient,
       license: licenseMock.createLicenseMock(),
-      experimentalFeatures: { prebuiltRulesCustomizationEnabled: true } as ExperimentalFeatures,
+      experimentalFeatures: {} as ExperimentalFeatures,
```

Comment:
Detection rules client doesn't depend on `experimentalFeatures` anymore, so this could be removed completely if we remove the `experimentalFeatures` parameter from the `createDetectionRulesClient` function.